packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -2,14 +2,19 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+apply from: "app_link_settings.groovy"
```

Comment:
Hi @reidbaker May I have some advice on how to better import other file? This works when running ./gradlew from android subproject directly, but does not work when running flutter build apk.

Reply:
reverted the change